.github/workflows/r-ci.yml

```diff
@@ -35,7 +39,11 @@ jobs:
         uses: lukka/get-cmake@latest
 
       - name: MkVars
-        run: mkdir ~/.R && echo "CXX17FLAGS=-Wno-deprecated-declarations -Wno-deprecated" > ~/.R/Makevars
+        run: |
+          set -e
+          mkdir ~/.R
+          echo "CXX17FLAGS=-Wno-deprecated-declarations -Wno-deprecated" > ~/.R/Makevars
+          echo "CCACHE=ccache" > ~/.R/Makevars
```

Comment:
It needs a bit more because of how R lays out files. I [once wrote about that](https://dirk.eddelbuettel.com/blog/2017/11/27/) and still copy the snippet for `~/.ccache/ccache.conf` 
around:

```
max_size = 5.0G
# important for R CMD INSTALL *.tar.gz as tarballs are expanded freshly -> fresh ctime
sloppiness = include_file_ctime
# also important as the (temp.) directory name will differ
hash_dir = false

Reply:
Also your second echo wants a `>>`